### PR TITLE
[Metricbeat] Update Couchbase module to 6.5 version

### PR DESCRIPTION
**Pull request by sayden**
## What does this PR do?
Changes the number format of some fields that seems to be returned in floating point while we expect them like integers.

It seems that some of the values returned by Couchbase are in floating point numbers. In fact, all the data is returned in JSON format so every single number might be a floating point number.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [x] Check that dashboards still works in v4.5, v5 and v6.5

## How to test this PR locally

Just execute the module normally. Unfortunately, the bug can't be reproduce easily (or at all)

## Related issues
Fixes: https://github.com/elastic/beats/issues/18949
Fixes: https://github.com/elastic/beats/issues/18595

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -231,6 +231,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix getting gcp compute instance metadata with partial zone/region in config. {pull}18757[18757]
 - Add missing network.sent_packets_count metric into compute metricset in googlecloud module. {pull}18802[18802]
 - Fix compute and pubsub dashboard for googlecloud module. {issue}18962[18962] {pull}18980[18980]
+- Update Couchbase to version 6.5 {issue}18595[18595] {pull}19055[19055]
```
Comment (jsoriano) [inline CHANGELOG.next.asciidoc:234]: Should compatibility notes and version used in integration tests also updated?

Comment (jsoriano) [inline CHANGELOG.next.asciidoc:234]: Add another changelog entry about fixing #18949?

Comment (sayden) [reply CHANGELOG.next.asciidoc]: Thank you very much! I knew I was forgetting something! I have updated the couchbase version of the Dockerfile and I'll update docs while CI is working

Comment (sayden) [reply CHANGELOG.next.asciidoc]: Done!

Comment (jsoriano) [reply CHANGELOG.next.asciidoc]: Thanks!

**Pull request merged**